<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCameraCalib</class>
 <widget class="QWidget" name="FormCameraCalib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormCameraCalib</string>
  </property>
  <widget class="QStackedWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>718</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="CameraWidgetCalib" name="page_1">
    <widget class="QPushButton" name="calib_1">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>660</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>标定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ezh_1">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>480</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QImageLineEdit" name="yz_1">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>540</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>包装速度：?</string>
     </property>
     <property name="text">
      <string>阈值：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yzjian_1">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>600</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yzjia_1">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>600</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="CameraWidgetCalib" name="page_2">
    <widget class="QPushButton" name="calib_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>660</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>标定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ezh_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>480</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yzjian_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>600</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QImageLineEdit" name="yz_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>540</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>包装速度：?</string>
     </property>
     <property name="text">
      <string>阈值：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yzjia_2">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>600</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="CameraWidgetJoint" name="page_3"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>718</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>718</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>718</y>
     <width>134</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>718</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraWidgetJoint</class>
   <extends>QWidget</extends>
   <header>CameraWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QImageLineEdit</class>
   <extends>QLineEdit</extends>
   <header>MWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CameraWidgetCalib</class>
   <extends>QWidget</extends>
   <header>CameraWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
